### ISSUE TITLE
Enable tabindex on grid

### ISSUE THREAD

**Issue by sidnc86** [techvision/waiable#4]
Enable tabindex property on the grid so that it is focusable. For this ensure that markup is generated as follows
```html
<div role=”grid” tabindex=”-1”>
  <div role=”row” tabindex=”-1”>
    <div role=”gridcell” tabindex=”0”>
      ...
    </div>
    <div role=”gridcell” tabindex=”-1”>
      ...
    </div>
  </div>
  ....
</div>
```
Note that care is taken to assign tabindex=”0” only to one gridcell that will be focused when user navigates the grid by pressing tab keys. The grid itself and rows are having tabindex=”-1” to prevent user focus. It should be ensured that only one gridcell has tabindex=”0” whereas the rest of the gridcells must have tabindex=”-1”.
This should be generated as part of markup generated as output of grid_for method.

**Comment by sidnc86** [comment]
Closing as focus is now switched towards basic content accessibility instead of building rich widgets.
